docs/src/main/asciidoc/kafka-streams.adoc

```diff
@@ -1112,6 +1112,107 @@ CMD ["./application", "-Dquarkus.http.host=0.0.0.0", "-Xmx32m"]
 Now start Docker Compose as described above
 (don't forget to rebuild the container images).
 
+== Kafka Streams Health Check
+
+If you are using the `quarkus-smallrye-health` extension, `quarkus-kafka-streams` will automatically add:
+
+* a readiness health check to validate that all topics declared in the `quarkus.kafka-streams.topics` property are created,
+* a liveness health check based on the Kafka Streams state.
+
+So when you access the `/health` endpoint of your application you will have information about the state of the KStreams and the available and/or missing topics.
+
+This is an example of `DOWN` status:
+[source, subs=attributes+]
+----
+curl -i http://aggregator:8080/health
+
+HTTP/1.1 503 Service Unavailable
+content-type: application/json; charset=UTF-8
+content-length: 454
+
+{
+    "status": "DOWN",
+    "checks": [
+        {
+            "name": "Kafka Streams state health check",  <1>
+            "status": "DOWN",
+            "data": {
+                "state": "CREATED"
+            }
+        },
+        {
+            "name": "Kafka Streams topic(s) health check",  <2>
```

Comment:
```suggestion
            "name": "Kafka Streams topics health check",  <2>
```
I will get rid of the already existing ones with `(s)`. It's less readable and doesn't bring much.